### PR TITLE
updated images and videos for better browser loading  

### LINKED ISSUE #973: Low Lighthouse score of Meshery.io
### Description
meshery.io is failing the Core Web Vitals Assessment

Mobile performance score is between 20-40

Check here:- https://pagespeed.web.dev/report?url=https%3A%2F%2Fmeshery.io%2F&form_factor=mobile

### Expected Behavior
The landing page should pass the Core Web Vitals Assessment

### Screenshots

Screenshot 2023-01-15 at 9 32 57 PM

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by saisuvanth**
**Description** 
Regarding the lighthouse issue, I have updated all images to webp and some videos to webm which are blocking the page rendering. Also achieved a significant improvement in lighthouse score ( The overall score is low because of not following best practices such as https in localhost). Score will increase in production build, because of _caching_ and _HTTPS_ usage.

This PR fixes #973 

**Before**

j

**After**

h

**Notes for Reviewers**

Can you please let me know how the website is deploying at production build. The overhead here is because of size of _CSS_ and _JS_ files which need browser caching configuration from serving side. 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@saisuvanth let's discuss this on websites call, add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) , if you would :)

**Comment by Shivam-AfA** [comment]
Woah! Great work @saisuvanth This is something really impactful that you have worked on here. Thanks for this.

**Comment by leecalcote** [comment]
merge conflicts...

**Comment by leecalcote** [comment]
@ayushthe1 please help move this PR forward.

**Review by ayushthe1 — CHANGES_REQUESTED** [review]
Thanks for the changes @saisuvanth .Could you also add `loading = lazy` to img tags where it's not been used. Also did you manually converted all the images to webp or used some sort of script ?

**Comment by vishalvivekm** [comment]
> Yeah okay will do that, and yeah I wrote a bash script for image conversion.

any update @saisuvanth ?

**Comment by saisuvanth** [comment]
@vishalvivekm  yep updated all images to load lazily. You can merge the branch safely.
Also I want to do some research on this issue for further improvement.

**Review by ayushthe1 — APPROVED** [review]
Thanks for the changes @saisuvanth :+1:

**Comment by saisuvanth** [comment]
@thisiskaransgit  Hey, can you check once I have merged and pushed into the same branch.

**Comment by Shivam-AfA** [comment]
@saisuvanth Still merge conflicts....

File: _includes/navigation.html [hunk 1]
```diff
@@ -29,11 +29,11 @@
 					{% for sublink in link.subitems %}
 					<li >
 						<div {% if sublink.link == page.url %} class="sub-nav-li sub-nav-li--active" {% else %} class="sub-nav-li" {% endif %}>
-								<img class="nav-img" src="{{sublink.img_src}}">
+								<img class="nav-img" loading="lazy" src="{{sublink.img_src}}">
```
Comment (Udit-takkar) [inline _includes/navigation.html:32]: I don't think we should not lazy load navigation because it is directly visible to user when the page loads

Comment (Chadha93) [reply _includes/navigation.html:32]: @saisuvanth ??

Comment (saisuvanth) [reply _includes/navigation.html:32]: Okay I'll change that into normal loading 
any fixes other than that?

Comment (thisiskaransgit) [reply _includes/navigation.html:32]: @saisuvanth, resolve the conflicts please

**Comment by Chadha93** [comment]
@saisuvanth Any update here? Let's close this PR, given the unaddressed feedback and the number of merge conflicts. Feel free to open new PR.

**Pull request closed without merging**